src/main/java/com/currencycloud/client/dirty/DirtyWatcherDeserializer.java

```diff
@@ -19,5 +19,6 @@ public E convert(E watched) {
     public static class Contact extends DirtyWatcherDeserializer<com.currencycloud.client.model.Contact> {}
     public static class Beneficiary extends DirtyWatcherDeserializer<com.currencycloud.client.model.Beneficiary> {}
     public static class Payment extends DirtyWatcherDeserializer<com.currencycloud.client.model.Payment> {}
+    public static class PaymentAuthorisation extends DirtyWatcherDeserializer<com.currencycloud.client.model.PaymentAuthorisation> {}
```

Comment:
No need to filter for changes as no PaymentAuthorisation objects are updated server-side